src/averageAges.js

```diff
@@ -54,8 +72,29 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
+function findMother(people, child) {
+  return people.some(mother => mother.name === child.mother);
+}
+
+function motherBorn(people, mother) {
```

Comment:
It will be better to name functions using verbs. For instance, if you will call this function getMotherBorn will be easier to understand a function's purpose.

Reply:
agree! we name functions as verbs, e.g. 'getMotherBorn'